app/helpers/stories_helper.rb

```diff
@@ -1,2 +1,13 @@
 module StoriesHelper
+  def status_label(story)
+    "<span class='story-status-badge #{story.status}'>#{story.status}</span>".html_safe
+  end
+
+  def status_color(story)
+    return "green" if @story.approved?
+
+    return "magenta" if @story.rejected?
+
+    return "orange" if @story.pending?
+  end
```

Comment:
If Rubocop is not complaining it's okay but I'm sure it will complain in the future because of the last explicit return.
Also if `pending` is the default we can remove the `if`.


```suggestion
  def status_color(story)
    return "green" if @story.approved?
    return "magenta" if @story.rejected?

    "orange"
  end
```